### ISSUE TITLE
Webpack 5 error

### ISSUE THREAD

**Issue by livelxw** [rohit-gohri/redocusaurus#25]
when call:
`docusaurus start`

it throws: 
`Module not found: Error: Can't resolve 'path' in './node_modules/@redocly/openapi-core/lib/config'`
`BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.`

docusaurus version:
`2.0.0-alpha.75`

**Comment by sibelius** [comment]
you need to polyfll node apis for webpack 5

check this https://gist.github.com/sibelius/24f63eef7f43b15dc73c4a0be11bbef8

**Comment by rohit-gohri** [comment]
I fixed polyfills in latest PR #40 but I'm getting this error now:

```
TypeError: Cannot read property 'split' of undefined
error building locale=en
Error: Failed to compile with errors.
    at /Users/rohitgohri/Projects/personal/redocusaurus/node_modules/@docusaurus/core/lib/webpack/utils.js:198:24
    at /Users/rohitgohri/Projects/personal/redocusaurus/node_modules/webpack/lib/MultiCompiler.js:550:14
    at processQueue (/Users/rohitgohri/Projects/personal/redocusaurus/node_modules/webpack/lib/MultiCompiler.js:488:6)
    at processTicksAndRejections (internal/process/task_queues.js:79:11)
```

Might be related to https://github.com/facebook/docusaurus/issues/3515 maybe as I'm using yarn workspaces now. Still investigating.

**Comment by rohit-gohri** [comment]
Fixed in v0.4.0
